fastly/block_fastly_service_v1_logging_ftp_test.go

```diff
@@ -265,12 +269,13 @@ resource "fastly_service_compute" "foo" {
     name       = "ftp-endpoint"
     address    = "ftp.example.com"
     user       = "user"
-	public_key = file("test_fixtures/fastly_test_publickey")
+	  public_key = file("test_fixtures/fastly_test_publickey")
     password         = "p@ssw0rd"
     path             = "/path"
     port             = 27
     gzip_level       = 3
-    timestamp_format = "%%Y-%%m-%%dT%%H:%%M:%%S.000"
+		timestamp_format = "%%Y-%%m-%%dT%%H:%%M:%%S.000"
+		message_type     = "classic"
```

Comment:
Looks like there are some whitespace issues here.

Reply:
It switched them to tabs which is why it got confused. Fixed